misc/life_mngr/s5_trigger.sh

```diff
@@ -0,0 +1,214 @@
+#! /bin/bash
```

Comment:
Not sure if strictly needed but we usually include the copyright and license information in our scripts.

Reply:
Where do all these `echo` messages end up? This script is called directly by `life_mngr` so I'm wondering how these will be exposed to the user?

Reply:
> Where do all these `echo` messages end up? This script is called directly by `life_mngr` so I'm wondering how these will be exposed to the user?

There are numerous use cases for the script. The user can execute the script directly and they can see the output in their shell. If it's called inside life_mngr, then they don't see the `echo` messages.

Reply:
ok, thanks for the clarification